.github/workflows/dev-cicd.yml

```
@@ -0,0 +1,93 @@
+      - closed
+
+  push:
+    branches: [ "dev" ]  # 병합 후 dev 브랜치로 푸시될 때 CD 실행
+
+jobs:
+  CI:
+    if: github.event_name == 'pull_request' && github.event.action != 'closed'  # PR이 닫히지 않았을 때 실행
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)